### PR TITLE
Create b.txt

### PR DESCRIPTION
**Pull request by DeekshithSN**

### CODE REVIEW

File: b.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+rtgyhj
+ghj
+vgbhnj
```
Comment (DeekshithSN) [inline b.txt:3]: fghjk

Comment (DeekshithSN) [reply b.txt:3]: this is ecgvbh

**Review by gauravksahni — APPROVED** [review]
approved

**Review by somnathbhusare — COMMENTED** [review]
done

**Pull request merged**